python/utilities/log_list/print_log_list.py

```diff
@@ -89,19 +87,27 @@ def print_formatted_log_list(json_log_list):
                 final_sth["tree_size"])
         print "-" * 80
 
-def run():
+
+def main(argv):
+    del argv  # Unused; all arguments provided via flags.
```

Comment:
Usually, what we do with unused flags like here is name them `unused_argv` (which shuts up the linter).

Reply:
My linter sadly does complain if I name it `unused_argv`. If I prefix it with an underscore, it's happy though.